accounts/models.py

```diff
@@ -254,3 +255,136 @@ class Message(models.Model):
         MessageType, on_delete=models.CASCADE,
         related_name='message_type'
     )
+
+
+# Revised models
+class Context(models.Model):
+    """
+    A context defines the level at which a role operates or an action is performed
+    within the system. Example: Organization, city, class levels. It includes a name,
+    description, and an order to define its level of hierarchy.
+
+    Attributes:
+        name (str): The name of the context.
+        description (str): A detailed description of the context.
+        order (int): The level of hierarchy for the context.
+
+    Meta:
+        ordering (list): The instances are ordered by their 'order' attribute in
+        ascending order.
+
+    Methods:
+        __str__(): Returns a string representation of the context in the format
+            'order - name'.
+    """
+    name = models.CharField(max_length=100, null=False, unique=True)
+    description = models.TextField(null=False)
+    order = models.IntegerField(null=False)  # Level of hierarchy
+
```

Comment:
should have a parent context also. for example, we will have state context with parent of all India context and various districts with parent context of their state context,

We will discuss this further.